### ISSUE TITLE
Url rewriting and download

### ISSUE THREAD

**Issue by dmaglio** [seblucas/cops#225]
i have enabled url rewriting and work good, the url is /dowload/... but i have error 404. i need to add other option in my config_local.php. now is
```php
<?php
    if (!isset($config))
        $config = array();
    $config['calibre_directory'] = '/mnt/calibre/';
    $config['cops_title_default'] = "COPS";
    $config['cops_use_url_rewriting'] = "1";
    $config['cops_template'] = 'bootstrap';
    $config['default_timezone'] = "Europe/Rome";
    $config['cops_language'] = 'it';
    $config['cops_x_accel_redirect'] = "X-Sendfile";
    $config['calibre_internal_directory'] = '/mnt/calibre/';
```

**Comment by aroundmyroom** [comment]
sometimes such issues can be related to the configuration of your webserver
Today I installed debian 8, installed php5, sqlite etc.. all was fine. I copied my 12.000 ebooks and whop .. working, but downloading .. oh oh .. 

Problem: apache2 by default does not allow a rewrite. Enable this module will not auto-fix this rewrite issue. Cause .. there is an feature called AllowOverride in the apache.conf which was not enabled on the folder yet. Reconfigure, restart the webserver and now Marvin on my iPAD can download the epub onto my iPAD .. 

Unfortunately you do not mention on what device or system you use cops so above could be helpful but maybe not
